plugins/config_manager.py

```diff
@@ -71,7 +78,13 @@ def configure(self):
                                      .format(' '.join([
                                          "--save", "--add-repo",
                                          "--dump", "--dump-variables",
-                                         "--enable", "--disable"])))
+                                         "--set-enabled", "--enable",
+                                         "--set-disabled", "--disable"])))
```

Comment:
You need to modify the tests in plugins-core/config-manager.feature to cope with this modified stderr message:
```Command line error: one of the following arguments is required: --save --add-repo --dump --dump-variables --set-enabled --enable --set-disabled --disable```
instead of original
```Command line error: one of the following arguments is required: --save --add-repo --dump --dump-variables --enable --disable```